### ISSUE TITLE
May we have testing ground-truth in raw data?

### ISSUE THREAD

**Issue by tianzhiliang** [qkaren/converse_reading_cmr#9]
Could you please release the ground-truth of the testing set in its raw data? 
We can access to raw data of training set, developing set, and testing set (https://drive.google.com/file/d/1U9F2qf7mA39bPpqcnY0VarnYDQ1ygS6e/view). But, the ground-truth of the testing set is hidden in raw data. We are required to create testing ground-truth from scratch by "make -j4 valid test refs", which takes a long time to download several large original reddit log files, and usually result in failure.

The best solution is to provide the completed testing set in raw data without hiding the ground-truth. Thanks!

**Comment by qkaren** [comment]
The test set is now under `evaluation/`

**Comment by tianzhiliang** [comment]
Thanks! Very helpful!
